feedbackreport/tests.py

```diff
@@ -125,4 +126,31 @@ def test_viewfeedback_negative(self):
             'session_id': session.session_key,
         })
 
-        self.assertNotEqual(json.loads(response.content)[0]['feedback_goal'], 'home page error')
\ No newline at end of file
+        self.assertNotEqual(json.loads(response.content)[0]['feedback_goal'], 'home page error')
+    
+    def test_editfeedback(self):
+        self.setup_account()
+        session = self.client.session
+        session['_auth_user_id'] = testemail
+        session.save()
+
+        self.client.post(add_feedback_url,json.dumps({
+            'session_id' : session.session_key,
+            'input_feedback_rating' : 5,
+            'input_feedback_goal' : 'i hope i will save money',
+            'input_feedback_text' : 'less impulsive expenses',
+            'input_feedback_text2' : 'very helpful and fun to do',
+            'input_feedback_comment' : 'i think the application has quite good features',
+
+        }),content_type=content_type)
+
+        response = self.client.post(edit_feedback_url,json.dumps({
+            'session_id': session.session_key,
+            'id': 1,
+            'input_feedback_rating' : 2,
+            'input_feedback_goal' : 'i hope i will save money more',
+            'input_feedback_text' : 'not less impulsive',
+            'input_feedback_text2' : 'not very helpful',
+            'input_feedback_comment' : 'not quite good features',
+        }),content_type=content_type)
+        
```

Comment:
The response is not used to check if the application has a status code of 200 which means it does not have errors. A quick fix can be added by adding the line below at the end of the test
```suggestion
        self.assertEqual(response.status_code, 200)
```